test/addons-abi/test_object/test_object.cc

```diff
@@ -1,4 +1,4 @@
-﻿#include <node_jsvmapi.h>
+#include <node_jsvmapi.h>
```

Comment:
There are few changes like this that I'm not sure what actually changed ?  Is it whitespace ?